bitrise.yml

```diff
@@ -287,150 +338,21 @@ workflows:
-    - custom-test-results-export@0:
-        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
+    - xcode-test-without-building@0:
+        title: UI Smoketest3
         inputs:
-        - search_pattern: "$BITRISE_SOURCE_DIR/xcodebuild.xcresult"
-        - test_name: SmokeTest4
+        - destination: platform=iOS Simulator,name=iPhone 14 Plus,OS=latest
+        - xcodebuild_options: '"-derivedDataPath" "/Users/vagrant/git/DerivedData" "COMPILER_INDEX_STORE_ENABLE=NO"
+            "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
+        - xctestrun: "Users/vagrant/git/DerivedData/Build/Products/Fennec_Enterprise_XCUITests_Smoketest4_iphonesimulator16.1-x86_64.xctestrun"
     - deploy-to-bitrise-io@2:
         run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
-  build_and_test_5_UnitTest:
-    steps:
-    - git::https://github.com/bitrise-steplib/bitrise-step-artifact-pull.git@main:
-        title: Pull artifacts
-        inputs:
-        - verbose: true
-        - artifact_sources: stage_1.*
-    - script@1.1:
-        title: Unzip
-        inputs:
-        - content: |
-            set -euxo pipefail
-
-            echo "$BITRISE_ARTIFACT_PATHS"
-
-            echo "-- unzip -- "
-            exec unzip "${BITRISE_ARTIFACT_PATHS}"
-            echo "show dir"
-            ls
-    - script@1.1:
-        title: Test without Building
-        inputs:
-        - content: |
-            #!/usr/bin/env bash
-            set -ex
-
-            ls -la
-            ls ./Users/vagrant/git/Tests/
-            cp -r ./Users/vagrant/git/Tests/* .
-            mv ./Users/vagrant/git/* .
-            ls -la
-
-            pwd
-            echo "-- test-without-building --"
-
-            xcodebuild -resultBundlePath "xcodebuild.xcresult" -derivedDataPath "/Users/vagrant/git/DerivedData" -destination "platform=iOS Simulator,name=iPhone 14,OS=latest" test-without-building -xctestrun "/Users/vagrant/git/DerivedData/Build/Products/Fennec_Enterprise_XCUITests_UnitTest_iphonesimulator16.1-x86_64.xctestrun"
-    - script@1.1:
-        title: Compress Result Bundle
-        is_always_run: true
-        inputs:
-        - content: |-
-            set -euxo pipefail
-            echo "-- compress --"
-
-            cd "${BITRISE_DEPLOY_DIR}/"
-            exec zip -0 -qr "${BITRISE_SOURCE_DIR}/TestResults.zip" "/Users/vagrant/git/xcodebuild.xcresult"
-    - deploy-to-bitrise-io@2:
         inputs:
-        - deploy_path: "${BITRISE_SOURCE_DIR}/TestResults.zip"
-    - custom-test-results-export@0:
-        inputs:
-        - search_pattern: "$BITRISE_SOURCE_DIR/xcodebuild.xcresult"
-        - test_name: Unit Test
-    - deploy-to-bitrise-io@2.0: {}
+        - pipeline_intermediate_files: '$BITRISE_XCRESULT_PATH:BITRISE_UITESTS_4_XCRESULT_PATH'
     meta:
       bitrise.io:
         stack: osx-xcode-14.1.x
         machine_type_id: g2.8core
-  just_build_fennec_to_get_warnings:
```

Comment:
I have removed this step. Since building for each scheme happens in first stage, the warning count and slack notification have been moved to the configure_build_fennec and configure_build_fennec_xcuitest accordingly